infection_monkey/system_info/windows_info_collector.py

```diff
@@ -27,6 +27,8 @@ def get_info(self):
         self.get_hostname()
         self.get_process_list()
         self.get_network_info()
+        self.get_azure_info()
         mimikatz_collector = MimikatzCollector()
-        self.info["credentials"] = mimikatz_collector.get_logon_info()
+        mimikatz_info = mimikatz_collector.get_logon_info()
```

Comment:
Any reason you took this out to a variable?

Reply:
Debugging, I'd rather leave it that way so we can observe before after in the credentials. Because if for some reason, we remove the get_azure_info call, the update call will fail.